lib/rubocop/cop/style/safe_navigation.rb

```diff
@@ -250,6 +252,10 @@ def add_safe_nav_to_all_methods_in_chain(corrector,
             break if ancestor == method_chain
           end
         end
+
+        def safeguarded?(receiver)
+          receiver.nil? || LOGIC_JUMP_KEYWORDS.include?(receiver.type)
```

Comment:
Won't this be slightly better as `receiver && LOGIC_JUMP_KEYWORDS.include?(receiver.type)`? I'm not sure this method is needed, though, as I think it's name is confusing.

Reply:
Agreed, removed that method and applied your suggestion.